projects/plugins/crm/ZeroBSCRM.php

```diff
@@ -23,6 +23,66 @@
 	exit;
 }
 
+// Enabling THIS will start LOGGING PERFORMANCE TESTS
+// NOTE: This will do for EVERY page load, so just add temporarily else adds rolling DRAIN on sys
+// define( 'ZBSPERFTEST', 1 );
+
+if ( ! defined( 'ZBS_ROOTFILE' ) ) {
+	define( 'ZBS_ROOTFILE', __FILE__ );
+	define( 'ZBS_PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
+	define( 'ZBS_ROOTDIR', basename( __DIR__ ) );
+	define( 'ZBS_ROOTPLUGIN', ZBS_ROOTDIR . '/' . basename( ZBS_ROOTFILE ) );
+	define( 'ZBS_LANG_DIR', basename( __DIR__ ) . '/languages' );
+}
```

Comment:
I've moved this further up on the page so it's defined before we try to call `ZBS_PLUGIN_DIR` for the autoloader.